x/did/types/codec.go

```diff
@@ -0,0 +1,54 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+	"github.com/ixofoundation/ixo-blockchain/x/did/exported"
+)
+
+func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
+	cdc.RegisterInterface((*exported.DidDoc)(nil), nil)
+
+	cdc.RegisterConcrete(&MsgAddDid{}, "did/AddDid", nil)
+	cdc.RegisterConcrete(&MsgAddCredential{}, "did/AddCredential", nil)
+
+	// TODO: https://github.com/ixofoundation/ixo-blockchain/issues/76
+	cdc.RegisterConcrete(&BaseDidDoc{}, "did/BaseDidDoc", nil)
+	//cdc.RegisterConcrete(DidCredential{}, "did/DidCredential", nil)
+	//cdc.RegisterConcrete(Claim{}, "did/Claim", nil)
```

Comment:
Remove comments

Reply:
These were here before, do we remove them?